src/instance.rs

```diff
@@ -164,6 +164,17 @@ impl<T> IntoPyPointer for Py<T> {
     }
 }
 
+// Native types `&T` can be converted to `Py<T>`
+impl<'a, T> std::convert::From<&'a T> for Py<T>
+where
+    T: AsPyPointer + PyNativeType,
```

Comment:
The impl for `PyObject` is:

    impl<'a, T> std::convert::From<&'a T> for PyObject
    where
        T: AsPyPointer,

Maybe here we also don't need the `PyNativeType` bound?

`PyObject` also has the same impl for `&'a mut T`, but I'm unsure whether we want that.

Reply:
This bound is not to allow `PyClassShell<T>` -> `Py<PyClassShell<T>>`

Reply:
Ah, seems reasonable :+1: